CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# How to contribute
+
+Whether you are looking to report a bug, propose a new feauture or simply getting involved in the project, this document is intended as a guide/reference for you!
+
+<a name="reporting_bugs"></a>
+
+## Reporting Bugs
+If you encounter any kind of issue with any of the Dyna&omega;o [repositories](https://github.com/dynawo), the first things to do is to ask for help creting an issue in the repository where the problem has been identified. If possible, explain in the most clear way the matter and try to associate one of the available labels to the issue. The github [repositories](https://github.com/dynawo) are the privileged environment to exchange informations.
+
+If you experince any difficulty to create an issue on one of the Dyna&omega;o repository, you can also send an e-mail to [rte-dynawo@rte-france.com](mailto:rte-dynawo@rte-france.com).
+
+## Creating a Pull Request
+If you wish to contribute to one of the Dyna&omega;o [repositories](https://github.com/dynawo) proposing a patch on a newly created issue or an existing one, please create a dedicated [git branch](#branch_creation). Once you consider you contribution ready for being merged in the main repository branch, please create a pull request on the Dyna&omega;o github repository, it will be reviewed by the development team as soon as possible. If any modification/discussion is need on your contribution, you will be notified on the related pull request. Notice that there is also the possibility to open a pull request in a draft mode.
```

Comment:
Maybe we should specify that the name of the branch is linked to the ticket number?